### DISCUSSION TITLE
Restricted to only 32 lists in GitHub? Are tags also to be restricted to only 32?

### DISCUSSION THREAD

**Discussion by meutrei** [community/community#21344] [Other Feature Feedback, Questions, & Ideas]
**Best GitHub joke ever:** Lists have appeared, but you only have 32 list!

We want to know when you are going to restrict tags to just 32 tags? 

**Remove those silly restrictions**, you think you can organize for quick access, but find you're limited to 32 lists. Congratulations GitHub :(

In addition, add the possibility of:
- Sublist
- Drag & Drop Repository from 1 list to another list
- Add labels to Repositories in list, e.g. Ubuntu, V2 Bug with jQuery, Need Python 3. 
All this for quick filtering and sorting of your Repositories within GitHub.

With the appearance of Lists in GitHub I thought I was getting rid of bookmarks, that I could save and sort everything inside GitHub, but GitHub restricted to 32 lists :)))))

**Comment by radar-blip** [comment]
It was 16 at first, in time, they may update again.
